searchlib/src/vespa/searchlib/features/fieldmatchfeature.cpp

```diff
@@ -18,35 +18,50 @@ using CollectionType = FieldInfo::CollectionType;
 
 namespace search::features {
 
+class FieldMatchExecutorSharedState : public Anything {
+private:
+    PhraseSplitterQueryEnv _splitter_env;
+    fieldmatch::ComputerSharedState _cmp_shared_state;
+public:
+    FieldMatchExecutorSharedState(const fef::IQueryEnvironment& query_env,
+                                  const fef::FieldInfo& field,
+                                  const fieldmatch::Params& params);
+    ~FieldMatchExecutorSharedState() override;
+    const PhraseSplitterQueryEnv& get_phrase_splitter_query_env() const { return _splitter_env; }
+    const fieldmatch::ComputerSharedState &get_field_match_computer_shared_state() const { return _cmp_shared_state; }
```

Comment:
Consider reducing verbosity of function name. get_computer_shared_state() should be sufficient.

Reply:
Done